enterprise/coderd/scim.go

```diff
@@ -40,6 +42,69 @@ func (api *API) scimVerifyAuthHeader(r *http.Request) bool {
 	return len(api.SCIMAPIKey) != 0 && subtle.ConstantTimeCompare(hdr, api.SCIMAPIKey) == 1
 }
 
+// scimServiceProviderConfig returns a static SCIM service provider configuration.
+//
+// @Summary SCIM 2.0: Service Provider Config
+// @ID scim-get-service-provider-config
+// @Produce application/scim+json
```

Comment:
Should be able to use this now
```suggestion
// @Summary SCIM 2.0: Service Provider Config
// @ID scim-get-service-provider-config
// @Security Authorization
// @Produce application/scim+json
```

Reply:
Oh wait I'm stupid, disregard

Reply:
Yup no auth for this endpoint :+1: